data-prep-pipeline/data_prep_pipeline/config.py

```diff
@@ -3,3 +3,5 @@
     "hgnc-gene-table-e250.3/hgnc_complete_set"  # This is needed for fusion data
 )
 oncokb_annotated_taiga_id = "oncokb-annotated-mutations-7e2e.17/oncokb_annotated"  # This is needed for driver events data
+
+portal_compounds_taiga_id = "compound-metadata-de37.30/PortalCompounds"
```

Comment:
Not really related to this PR, but a passing thought: Updating this config.py should probably go into the release instructions